CRM/Mailing/BAO/Mailing.php

```diff
@@ -3173,10 +3173,11 @@ public static function getWorkflowFieldPerms() {
   }
 
   /**
-   * Whitelist of possible values for the entity_table field
+   * White-list of possible values for the entity_table field.
+   *
    * @return array
    */
-  public static function mailingGroupEntityTables($context = NULL) {
```

Comment:
Actually it looks like it was @colemanw who edited that fn so the var is not removed without removing it - any reason C?